packages/backend/src/routers/projectRouter.ts

```diff
@@ -377,3 +377,76 @@ projectRouter.get(
         }
     },
 );
+
+projectRouter.get('/projectAccess', isAuthenticated, async (req, res, next) => {
+    try {
+        const results = await projectService.getProjectAccess(
+            req.user!,
+            req.params.projectUuid,
+        );
+        res.json({
+            status: 'ok',
+            results,
+        });
+    } catch (e) {
+        next(e);
+    }
+});
+
+projectRouter.post(
+    '/projectAccess',
+    isAuthenticated,
+    async (req, res, next) => {
+        try {
+            const results = await projectService.createProjectAccess(
+                req.user!,
+                req.params.projectUuid,
+                req.body,
+            );
+            res.json({
+                status: 'ok',
+                results,
+            });
+        } catch (e) {
+            next(e);
+        }
+    },
+);
+projectRouter.patch(
```

Comment:
The last two functions I feel should either be:

`POST /projects/{projectUuid}/access` and `PATCH /projects/{projectUuid}/access/{policyUuid}` (so adding a new Uuid) - this leaves space for access records to allow group names instead of a user etc.

OR

`PUT /projects/{projectUuid}/userAccess/{userUuid}` - and make it user specific. 

I found the `PATCH` without any path parameters a bit counter intuitive because the user has to implicitly understand that the resources are unique on the project-user combination (I saw that in the table definition)

What do you think?

Reply:
Yeah, you are right,  we can use `userUuid` like I do on `delete`